packages/lenovolegionlinux-git/lenovolegionlinux-git.pacscript

```diff
@@ -0,0 +1,75 @@
+_pkgname="lenovolegionlinux"
+pkgname="${_pkgname}-git"
+pkgver="1.c1e9b55"
+pkgdesc="LenovoLegionLinux (LLL) brings additional drivers and tools for Lenovo Legion series laptops to Linux. PLEASE READ THE REPO BEFORE INSTALL THIS PACKAGE!!!"
+arch=("amd64")
+url='https://github.com/johnfanv2/LenovoLegionLinux'
+maintainer=("MrDuartePT <gonegrier.duarte@gmail.com>")
+license=('GPL-2.0-only')
+
+depends=(
+  python3
+  python3-argcomplete
+  python3-yaml
+  python3-pyqt6
+  polkitd
+)
+
+pacdep=(python3-darkdetect)
+
+makedepends=(
+  git
+  python3-build
+  python3-installer
+  python3-setuptools
+  python3-wheel
+  libinih-dev
+)
+
+optdepends=("lenovolegionlinux-dkms-git: DKMS module")
+
+replaces=("${_pkgname}")
+
+source=(
+  "${_pkgname}::git+https://github.com/johnfanv2/LenovoLegionLinux"
+  "https://raw.githubusercontent.com/benhoyt/inih/refs/heads/master/ini.c"
+)
+sha256sums=('SKIP')
```

Comment:
```suggestion
license=('GPL-2.0-only')
depends=(
  "python3"
  "python3-argcomplete"
  "python3-yaml"
  "python3-pyqt6"
  "polkitd"
)
pacdep=("python3-darkdetect")
makedepends=(
  ?git"
  "python3-build"
  "python3-installer"
  "python3-setuptools"
  "python3-wheel"
  "libinih-dev"
)
optdepends=("lenovolegionlinux-dkms-git: DKMS module")
replaces=("${_pkgname}")
source=(
  "${_pkgname}::https://github.com/johnfanv2/LenovoLegionLinux.git"
  "https://raw.githubusercontent.com/benhoyt/inih/refs/heads/master/ini.c"
)
sha256sums=('SKIP')
```

Can you clamp `ini.c` to a specific version so you can use `sha256sums` on it?

Secondly for `optdepends`, we currently cannot have pacstall packages in it. The only place where pacstall packages can go is `pacdeps`. We're working it fixing that though.